crates/rpc/rpc/src/eth/api/server.rs

```diff
@@ -180,11 +181,84 @@ where
 
     async fn fee_history(
         &self,
-        _block_count: U256,
-        _newest_block: BlockNumber,
+        block_count: u64,
+        newest_block: BlockNumber,
```

Comment:
this block type isn't actually correct.
it should be the BlockNumber enum and not the u64 type alias.
this alias is a bit unfortunate imo

Reply:
agree, changed to `BlockId` that's already used in a couple of other places in API